bindings/pyroot/pythonizations/test/CMakeLists.txt

```
@@ -191,6 +191,6 @@ ROOT_ADD_PYUNITTEST(pyroot_memory memory.py)
 # rbatchgenerator tests
 # TODO: We currently do not support TensorFlow for Python >= 3.12 (see requirements.txt)
 # Update here once that is fixed.
-if (NOT MSVC AND Python3_VERSION VERSION_LESS 3.12)
+if (NOT MSVC AND Python3_VERSION VERSION_LESS 3.12 AND tmva)
```

```suggestion
if (NOT MSVC AND tmva)
```
As discussed already by @lmoneta: the tensorflow dependency is already encoded in `PYTHON_DEPS`, and if TensorFlow is not installed as per our `requirements.txt`, then the test will be disabled anyway. So any Python version check is redundant.

I would suggest to not check any Python version and also remove the TODO above. In fact, as it is now, we will probably forget to re-enable the test once we support TensorFlow for Python >= 3.12... If this is only encoded in the `requirements.txt`, things are easier.

---

Thanks for the suggestion @guitargeek, I have removed the Python version check now.